performance-tests/bench/legacy_plotting/README

```diff
@@ -1,9 +1,39 @@
 To generate new data and image files based on a scoreboard test run directory:
 
-./generate-test-results.sh <TEST_RUN_DIR>
+1) Set up your environment by sourcing $DDS_ROOT/setenv.sh and $BENCH_ROOT/set_bench_env.sh
```

Comment:
It would be helpful to say where the OpenDDS installation on Centipede is located.

Reply:
Details of any servers that may be internal to OCI are not in scope for public documentation.

Reply:
I agree, I don't want to put too many details of centipede in here, but I also don't think it's wrong to mention that centipede is where we currently run these tests.